packages/react-sdk/docusaurus/docs/React/02-guides/05-call-types.mdx

```diff
@@ -3,154 +3,6 @@ id: configuring-call-types
 title: Call Types
 ---
 
-:::warning
+import CallTypesPage from '../../../shared/video/_call-types.md';
```

Comment:
Is this file already available in the shared folder?

Reply:
Oh I see it 👍🏻 https://github.com/GetStream/stream-chat-docusaurus-cli/blob/staging/shared/video/_call-types.md